.github/workflows/ci.yaml

```diff
@@ -42,8 +42,6 @@ jobs:
     steps:
       - name: Checkout branch
         uses: actions/checkout@v3
-        with:
```

Comment:
Would we also need to remove this for all of the respective integration tests?

Reply:
I'd say so

Reply:
Good catch!